src/tls.c

```diff
@@ -7159,13 +7226,17 @@ static const char* OQS_ID2name(int id)
  */
 static int TLSX_KeyShare_GenOqsKey(WOLFSSL *ssl, KeyShareEntry* kse)
 {
-    int ret = -1;
+    int ret = 0;
     const char* algName = NULL;
     OQS_KEM* kem = NULL;
     byte* pubKey = NULL;
     byte* privKey = NULL;
+    KeyShareEntry *ecc_kse = NULL;
+    int oqs_group;
```

Comment:
Might consider initializing these, since you don't have return code checking on `findEccOqs`.

Reply:
done...here and everywhere else.  Please see next commit.